websites/D/dizipal/presence.ts

```diff
@@ -0,0 +1,147 @@
+const presence = new Presence({
+		clientId: "1212379206521192458",
+	}),
+	strings = presence.getStrings({
+		play: "general.playing",
+		pause: "general.paused",
+		viewShow: "general.viewShow",
+		viewMovie: "general.buttonWatchMovie",
+		viewEpisode: "general.buttonViewEpisode",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	pages: {
+		[k: string]: string;
+	} = {
+		"/": "Anasayfaya Göz Atıyor",
+		"/filmler": "Filmlere Göz Atıyor",
+		"/seri-filmler": "Seri Filmlere Göz Atıyor",
+		"/diziler?kelime=&durum=&tur=26&type=": "Anime Dizilerine Göz Atıyor",
+		"/diziler": "Dizilere Göz Atıyor",
+		"/koleksiyon/asya-dizileri": "Asya Dizilerine Göz Atıyor",
+		"/diziler/son-bolumler": "Son Dizi Bölümlerine Göz Atıyor",
+		"/koleksiyon/": "Koleksiyonuna Göz Atıyor",
+		"/bildirimler": "Bildirimlerine Göz Atıyor",
+		"/izlediklerim": "İzlediklerine Göz Atıyor",
+		"/izleyeceklerim": "İzleyeceklerine Göz Atıyor",
+		"/takip-ettiklerim": "Takip Ettiklerine Göz Atıyor",
+		"/ayarlar": "Ayarlarına Göz Atıyor",
+	};
+
+const enum Assets {
+	Logo = "https://i.imgur.com/CMzeCH3.png",
+}
+
+interface IframeData {
+	duration: number;
+	currentTime: number;
+	paused: boolean;
+}
+
+let video: IframeData;
+presence.on("iFrameData", (data: IframeData) => {
+	video = data;
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, href } = document.location,
+		seriesPageCheck =
+			document.querySelector(
+				"#container > div.popup-inner.auto > div.cover > div > a:nth-child(1)"
+			) === null;
+
+	for (const [key, value] of Object.entries(pages)) {
+		if (pathname.includes("/dizi/") && seriesPageCheck) {
+			const stringsData = await strings,
+				seriesTitle = document
+					.querySelector("#container")
+					.querySelector("h5")?.textContent,
+				episode = document.querySelector(
+					"#container > div.episode-head > div:nth-child(1) > header > h2 > a"
+				)?.textContent,
+				seriesUrl = document.querySelector<HTMLAnchorElement>(
+					"#container > div.episode-head > div:nth-child(1) > header > h2 > a"
+				)?.href;
+
+			if (seriesTitle && !episode) {
+				presenceData.details = `${seriesTitle} Adlı Diziye Göz Atıyor.`;
+
+				presenceData.buttons = [
+					{
+						label: `${stringsData.viewShow} ${seriesTitle}`,
```

Comment:
@Bas950 not sure if we want variables in button labels?
```suggestion
						label: `${stringsData.buttonViewShow} ${seriesTitle}`,
```
otherwise if not:
```suggestion
						label: stringsData.buttonViewShow,
```

Reply:
yea we don't just the .buttonViewShow not the title next to it